### DISCUSSION TITLE
What does noise scale, chunk seconds and Pad seconds do?

### DISCUSSION THREAD

**Discussion by elchupacabrinski** [voicepaw/so-vits-svc-fork#190] [Q&A] [answered]
Please see title, thanks :)

**Comment by 34j** [comment, marked as answer]
Noise scale is a parameter that represents the magnitude of the noise component. I am not sure what happens if you change it, and I don't have enough information.
https://github.com/34j/so-vits-svc-fork/blob/main/src/so_vits_svc_fork/models.py#L135-L135
The other two parameters are for librosa.split.

**Comment by vertexgamer** [comment]
It might just be my immagination, but i found generated voices to have a bit less artifact and sound cleaner when the noise was in the 0.10 range
